LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/size-adjust-unicode-range-system-fallback-ref.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE HTML>
+<meta charset="utf-8">
+<link rel="author" href="mailto:changseok@webkit.org">
+<title>Tests that the size-adjust descriptor does not affect system fallback font size.</title>
+<style>
+@font-face {
+  font-family: large-font;
+  src: local(Ahem), url(/fonts/Ahem.ttf);
+  size-adjust: 1000%;
+  unicode-range: U+0020;
+}
+
+.space {
+  font-family: large-font;
+}
+
+.ref {
+  font-family: sans-serif;
```

Comment:
It might be good to explicitly define sans-serif as default in the test itself, since we expect `あ` to be rendered with that, or even better, a non-generic font-family.

Reply:
Thanks for your comment. We are importing this test from the WPT. Do you intend to modify it nevertheless?

> It might be good to explicitly define sans-serif as default in the test itself

Do you mean `font-family: large-font, sans-serif;` in size-adjust-unicode-range-system-fallback.html?